### PR TITLE
Azure Bastion

### PR DESCRIPTION
**Pull request by Gordonby**
## PR Summary

Added capability to deploy Azure Bastion to the networking bicep. Handy for private clusters.

- added playwright library to write more advanced tests, specifically to add coverage tests for the new Bicep UI in the Wizard. The library is called **playwright-expect** and I have referenced it in the contribution guide.
- added PSRule to the Regression workflow to facilitate a `Well Architected` params file example
- added a `Well Architected` params file to iterate on
- bumped the default Kubernetes version used by the bicep files
- changed the default custom network address ranges to be more size optimised

## PR Checklist

- [x] PR has a meaningful title
- [x] Summarized changes
- [x] This PR is ready to merge and is not **Work in Progress**
- [ ] Link to a filed issue

### CODE REVIEW

**Comment by Gordonby** [comment]
Tested this out by adding a jumpbox and using bastion to do a get nodes on the private cluster

image

File: helper/src/config.json [hunk 1]
```diff
@@ -438,6 +439,7 @@
                                 }
                             ],
                             "azurepolicy": "deny",
+                            "bastion": true,
```
Comment (khowling) [inline helper/src/config.json:442]: This looks like its still in the addons section?

Comment (khowling) [reply helper/src/config.json:442]: Also, if it goes into one of the security cards, it needs to go into all 3.   For example, If the user clicks from high security to no security, the change should be reversed out if possible

**Review by khowling — CHANGES_REQUESTED** [review]
Please see inline comments

File: helper/src/config.json (hunk 1 above)
Comment (Gordonby) [reply helper/src/config.json]: Good spot 👍 
Moved it.

**Review by khowling — APPROVED** [review]

**Pull request merged**